packages/orchestrator-ui-components/package.json

```diff
@@ -54,13 +54,14 @@
         "@testing-library/react-hooks": "^8.0.1",
         "@testing-library/user-event": "^14.4.3",
         "@types/invariant": "^2.2.33",
-        "@types/jest": "^27.4.0",
+        "@types/jest": "^29.5.8",
         "@types/object-hash": "^2.1.0",
         "@types/scroll-into-view": "^1.16.3",
         "esbuild": "^0.14.10",
         "esbuild-jest": "^0.5.0",
-        "jest": "^27.4.7",
-        "jest-watch-typeahead": "^1.0.0",
+        "jest": "^29.7.0",
+        "jest-watch-typeahead": "^2.2.2",
+        "jest-environment-jsdom": "^29.7.0",
```

Comment:
was this dependency needed in the end?

Reply:
Yes. Because of the Jest upgrade this is now not included by default anymore